WordPress/wordpress-playground
# Add network error message

Fixes #1205

## What is this PR doing?

It adds an error message for blueprint network errors.

## What problem is it solving?

It will help users identify the root cause of blueprints failing.

## How is the problem addressed?

By returning a custom error message with debug instructions.

## Testing Instructions

- Checkout this branch
- [Open Playground with this blueprint](http://localhost:5400/website-server/#{%20%22features%22:%20{%20%22networking%22:%20true%20},%20%22steps%22:%20[%20{%20%22step%22:%20%22installPlugin%22,%20%22pluginZipFile%22:%20{%20%22resource%22:%20%22url%22,%20%22url%22:%20%22https://github.com/helgatheviking/simple-user-listing-demo-avatars/releases/download/1.0.0/simple-user-listing-demo-avatars.1.0.0.zip%22%20},%20%22options%22:%20{%20%22activate%22:%20true%20}%20}%20]%20})
- Open the browser console and see the new error 

**Note**: The error is currently displayed multiple times because of uncaught promises somewhere in the code.
